src/main/java/wooteco/subway/domain/path/Path.java

```diff
@@ -1,43 +1,41 @@
 package wooteco.subway.domain.path;
 
-import org.jgrapht.Graph;
-import org.jgrapht.graph.DefaultWeightedEdge;
-import org.jgrapht.graph.WeightedMultigraph;
+import wooteco.subway.domain.path.factory.PathFactory;
 import wooteco.subway.domain.section.Section;
 import wooteco.subway.domain.station.Station;
 
 import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
 
 public class Path {
-    private final Graph<Long, DefaultWeightedEdge> graph;
-    private final PathFindingStrategy pathFindingStrategy;
+    private final List<Section> sections;
+    private final List<Station> stations;
 
-    public Path(List<Station> stations, List<Section> sections, PathFindingStrategy pathFindingStrategy) {
-        this.graph = generateGraph(stations, sections);
-        this.pathFindingStrategy = pathFindingStrategy;
+    public Path(List<Section> sections, List<Station> stations) {
+        this.sections = sections;
+        this.stations = stations;
```

Comment:
```suggestion
    private Path(List<Section> sections, List<Station> stations) {
        this.sections = sections;
        this.stations = stations;
```
정적팩토리로 사용을 유도하시려면 private으로 막는게 좋겠네요~